### PR TITLE
0.16.9 Release Notes

### PR DESCRIPTION
**Pull request by paolaperaza**
@vishwas-astro Want to take a quick peek at this?

Based these on the following GH issues with the `0.16.9` label: https://github.com/astronomer/issues/issues?q=is%3Aissue+label%3A0.16.9+is%3Aclosed

### CODE REVIEW

File: enterprise/next/09_resources/01_release-notes.md [hunk 1]
```diff
@@ -10,11 +10,60 @@ As of Astronomer v0.16, Astronomer releases will be made generally available to
...
+- AC 1.10.10-5 ([Changelog](https://github.com/astronomer/ap-airflow/blob/master/1.10.10/CHANGELOG.md))
+- AC 1.10.7-15 ([Changelog](https://github.com/astronomer/ap-airflow/blob/master/1.10.7/CHANGELOG.md))
+- AC 1.10.5-11 ([Changelog](https://github.com/astronomer/ap-airflow/blob/master/1.10.5/CHANGELOG.md))
+
+
+These patch releases include a fix to an identified security issue listed [here](https://www.astronomer.io/docs/ac/v1.10.10/get-started/security/).
+
+#### Bug Fixes and Improvements
+
+Two additional bugs were addressed in v0.16.9 -
```
Comment (danielhoherd) [inline enterprise/next/09_resources/01_release-notes.md:52]: Should this be changed to:
```suggestion
Two additional bugs were addressed in Astronomer v0.16.9:
```

Comment (paolaperaza) [reply enterprise/next/09_resources/01_release-notes.md:52]: @danielhoherd Yea, I felt like I was being a bit too repetitive but I think the changes you suggested would be better for consistency. Fixed!

**Review by paolaperaza — COMMENTED** [review]
Thank you for these typo fixes @danielhoherd !

**Review by danielhoherd — APPROVED** [review]

**Comment by paolaperaza** [comment]
Thanks @danielhoherd. In case you want to review these, I actually just added some changes to our existing ["Airflow Versioning"](https://www.astronomer.io/docs/enterprise/v0.16/customize-airflow/airflow-versioning/) doc as well reflecting support for AC 1.10.12 as of Astronomer v0.16.9.

(1.10.12 support was already released to Cloud, so you'll see 1.10.12 listed in the Cloud version already).

File: enterprise/next/09_resources/01_release-notes.md [hunk 2]
```diff
@@ -10,11 +10,58 @@ As of Astronomer v0.16, Astronomer releases will be made generally available to
...
+- A new `on_kill` method that ensures a KubernetesPodOperator task is killed when it's cleared in the Airflow UI ([commit]((https://github.com/apache/airflow/commit/ce94497cc) ))
+- Ability to define a custom XCom class ([commit]((https://github.com/apache/airflow/pull/8560)))
+- Support for grabbing Airflow configs with sensitive data from Secret Backends ([commit]((https://github.com/apache/airflow/pull/9645)))
+- Support for AirfowClusterPolicyViolation support in Airflow local settings ([commit](https://github.com/apache/airflow/pull/10282)).
+
+For a detailed breakdown of all changes, refer to the [AC 1.10.12 Changelog](https://github.com/astronomer/ap-airflow/blob/master/1.10.12/CHANGELOG.md). For instructions on how to upgrade to 1.10.12 on Astronomer, refer to ["Airflow Versioning"](https://www.astronomer.io/docs/cloud/stable/customize-airflow/airflow-versioning/).
+
+> **Note:** AC 1.10.12 will be the _last_ version to support an Alpine-based image. In an effort to standardize our offering and optimize for reliability, we'll exclusively build, test and support Debian-based images staring with AC 1.10.13. A guide for how to migrate from Alpine to Debian coming soon.
```
Comment (vishwas-astro) [inline enterprise/next/09_resources/01_release-notes.md:37]: type -- should be starting with instead of staring with

**Comment by vishwas-astro** [comment]
Looks good. Just one typo.

**Review by vishwas-astro — APPROVED** [review]

File: enterprise/next/09_resources/01_release-notes.md (hunk 2 above)
Comment (paolaperaza) [reply enterprise/next/09_resources/01_release-notes.md]: Good catch, thanks @vishwas-astro - fixed in all 3 release notes files.

**Pull request merged**